package-lock.json

```diff
@@ -1,8298 +1,8 @@
 {
   "name": "firebase-functions-test",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
This seems like a regression - you may want to regenerate this with npm 8

Reply:
ah yeah I was jumping around node versions a bit. Updated!

Reply:
And by updated I mean re-generated.